sentry_kafka_management/scripts/brokers.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+from typing import Sequence
+
+from sentry_kafka_management.actions.brokers import (
+    describe_broker_configs as describe_broker_configs_action,
+)
+from sentry_kafka_management.brokers import YamlKafkaConfig
+
+
+def describe_broker_configs(argv: Sequence[str] | None = None) -> int:
+    """Returns a list of topics"""
+    parser = argparse.ArgumentParser(
+        description="List Kafka topics using configuration file",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog="""
+Examples:
+  %(prog)s -c config.yml
+  %(prog)s -c config.yml -n my-cluster
+  %(prog)s --config config.yml --cluster production
+        """,
+    )
+
+    parser.add_argument(
+        "-c", "--cluster-config", required=True, help="Path to the cluster YAML configuration file"
+    )
+
+    parser.add_argument(
+        "-t", "--topic-config", required=True, help="Path to the topics' YAML configuration file"
+    )
+
+    parser.add_argument(
+        "-n",
+        "--cluster",
+        help="Name of the cluster to query (uses first available if not specified)",
+    )
```

Comment:
These are mostly the same parameters we use in the `topics` script but not exactly: `--config` there and `--cluster-config` here. 
Please let's put the common arguments in common and make it consistent. In other words add a function that declares and extracts the common arguments we can use in all scripts.